Api/app/controllers/admin/requests_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Admin
+  # app/controllers/admin/requests_controller.rb
+  class RequestsController < Admin::AdminController
+    before_action :set_request, only: :update
+
+    def index
+      render json: Request.manager
+    end
+
+    def update
+      @request.update(status: params[:status])
+      if @request.save
+        render json: @request, status: :ok
+      else
+        render json: @request.errors, status: :bad_reques
```

Comment:
Isn't it a `:bad_request`? or is it ok like this `:bad_request`?

Reply:
```suggestion
        render json: @request.errors, status: :bad_request
```